### PR TITLE
Info message when STORAGE_DRIVER is missing.

### PR DESCRIPTION
**Pull request by shishir-a412ed**
Signed-off-by: Shishir Mahajan <shishir.mahajan@redhat.com>

### CODE REVIEW

**Comment by shishir-a412ed** [comment]
ping @rhvgoyal @rhatdan

File: container-storage-setup.sh [hunk 1]
```diff
@@ -1047,8 +1047,7 @@ setup_storage() {
   local current_driver
 
   if [ "$STORAGE_DRIVER" == "" ];then
```
Comment (rhatdan) [inline container-storage-setup.sh]: Can you just change this line to 
if [ -z "$STORAGE_DRIVER ];

**Comment by rhvgoyal** [comment]
Actually, this is intentional. We support passing `STORAGE_DRIVER=""` so that if somebody wants to test with loop devices intentionally, docker-storage-setup can exit without setting up any storage.

**Comment by rhvgoyal** [comment]
So you might want to improve message but exiting with code 0 is intentional.

**Comment by rhatdan** [comment]
What should the message say?  

STORAGE_DRIVER not set, no storage will be configured. You must specify STORAGE_DRIVER if you want c-s-s to configure storage."

**Comment by rhvgoyal** [comment]
How about getting rid of word c-s-s? rest looks fine.

`STORAGE_DRIVER not set, no storage will be configured. You must specify STORAGE_DRIVER if you want to configure storage."`

**Comment by shishir-a412ed** [comment]
@rhvgoyal PTAL.

**Comment by rhatdan** [comment]
@rh-atomic-bot r+ a548498

**Pull request closed without merging**